tests/v2/validation/envvars/build_versions_test.go

```diff
@@ -0,0 +1,102 @@
+//go:build (infra.any || cluster.any || sanity || validation) && !stress && !extended
+
+package envvars
+
+import (
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"testing"
+
+	"github.com/rancher/shepherd/clients/rancher/catalog"
+	"github.com/rancher/shepherd/extensions/charts"
+
+	"github.com/rancher/shepherd/extensions/clusters"
+)
+
+type BuildVersionsTestSuite struct {
```

Comment:
Previously, there was a discussion about reducing the types defined in the global struct. Can we limit it to just these and move the rest within the individual tests?
```
client *rancher.Client
session *session.Session
cluster *management.Cluster
```

Also, looks like steveClient isn't used anywhere.

Reply:
removed, thank you!